src/platform/EFR32/BLEManagerImpl.cpp

```diff
@@ -564,61 +592,81 @@ CHIP_ERROR BLEManagerImpl::ConfigureAdvertisingData(void)
         snprintf(mDeviceName, sizeof(mDeviceName), "%s%04" PRIX32, CHIP_DEVICE_CONFIG_BLE_DEVICE_NAME_PREFIX, (uint32_t) 0);
 
         mDeviceName[kMaxDeviceNameLength] = 0;
+        mDeviceNameLength                 = strlen(mDeviceName);
```

Comment:
I guess we can do that, though we could also just use the return value of `snprintf` to compute this...

Reply:
@bzbarsky-apple true, we're planning to optimize our example in the next few days so this will likely get fix.